### PR TITLE
[fontdiff] Resize renderings to allow comparisons

### LINKED ISSUE #162: Make fontdiff compare renderings even if they aren't the same size
https://github.com/googlei18n/nototools/issues/158#issuecomment-175747707 suggests comparison is abandoned if the renders aren't the same size. For things like a ligature that has dropped part of it's contents ("fi" now renders just "i") that isn't ideal.

Resize the image to be able to provide visual comparison. Maybe just add whitespace?

### PR DESCRIPTION
**Pull request by jamesgk**
Should close #162

### CODE REVIEW

**Comment by dougfelt** [comment]
LGTM

File: nototools/shape_diff.py [hunk 1]
```diff
@@ -116,6 +118,12 @@ def find_rendered_diffs(self, font_size=24, stats=None):
...
                 '--features=%s' % ','.join(features), path_b, text])
+
+            img_info = subprocess.check_output(['identify', a_png]).split()
+            assert img_info[0] == a_png and img_info[1] == 'PNG'
+            subprocess.call([
+                'convert', '-gravity', 'center', '-background', 'black',
+                '-extent', img_info[2], b_png, b_png])
             with open(diffs_filename, 'a') as ofile:
                 subprocess.call(
                     ['compare', '-metric', 'AE', a_png, b_png, cmp_png],
```
Comment (davelab6) [inline nototools/shape_diff.py]: I suggest using the comparison command I quoted in https://github.com/googlei18n/nototools/issues/162#issuecomment-175885431

Comment (jamesgk) [inline nototools/shape_diff.py]: I'll use that for actually saving comparison images; this is just used for getting the absolute error metric.

Comment (davelab6) [inline nototools/shape_diff.py]: Oh I see! Sorry I was just reading the code (badly! :)

**Pull request merged**